pkg/bitcoin/electrum/electrum.go

```diff
@@ -566,6 +566,38 @@ func (c *Connection) getScriptMempool(
 	return convertedItems, nil
 }
 
+// EstimateSatPerVByteFee returns the estimated sat/vbyte fee for a
+// transaction to be confirmed within the given number blocks.
+func (c *Connection) EstimateSatPerVByteFee(blocks uint32) (int64, error) {
+	// According to Electrum protocol docs, the returned fee is BTC/KB.
+	btcPerKbFee, err := requestWithRetry(
+		c,
+		func(
+			ctx context.Context,
+			client *electrum.Client,
+		) (float32, error) {
+			return client.GetFee(ctx, blocks)
```

Comment:
Electrum documentation [says](https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-estimatefee) that `blockchain.estimatefee` is deprecated since version 1.4.2. Do you know what is the reason and what should be used instead (cc @nkuba)?

Reply:
This was a good decision but can we just drop a TODO above the `client.GetFee` call to warn about the deprecation?